watertap/costing/units/gac.py

```diff
@@ -28,136 +28,81 @@ class ContactorType(StrEnum):
-    )
-    blk.pres_contactor_cost_coeff_3 = pyo.Var(
-        initialize=0.110592,
-        units=pyo.units.USD_2020 * (pyo.units.m**3) ** -3,
-        doc="GAC steel pressure contactor polynomial cost coefficient 3",
-    )
-    blk.grav_contactor_cost_coeff_0 = pyo.Var(
-        initialize=75131.3,
-        units=pyo.units.USD_2020,
-        doc="GAC concrete gravity contactor polynomial cost coefficient 0",
-    )
-    blk.grav_contactor_cost_coeff_1 = pyo.Var(
-        initialize=735.550,
-        units=pyo.units.USD_2020 * (pyo.units.m**3) ** -1,
-        doc="GAC concrete gravity contactor polynomial cost coefficient 1",
-    )
-    blk.grav_contactor_cost_coeff_2 = pyo.Var(
-        initialize=-1.01827,
-        units=pyo.units.USD_2020 * (pyo.units.m**3) ** -2,
-        doc="GAC concrete gravity contactor polynomial cost coefficient 2",
+        doc="number of off-line redundant GAC contactors in parallel",
     )
-    blk.grav_contactor_cost_coeff_3 = pyo.Var(
-        initialize=0,
-        units=pyo.units.USD_2020 * (pyo.units.m**3) ** -3,
-        doc="GAC concrete gravity contactor polynomial cost coefficient 3",
+    blk.regen_frac = pyo.Var(
+        initialize=0.70,
+        units=pyo.units.dimensionless,
+        doc="fraction of spent GAC adsorbent that can be regenerated for reuse",
     )
+
+    # ---------------------------------------------------------------------
+    # correlation reference points
+
     blk.bed_mass_max_ref = pyo.Var(
         initialize=18143.7,
         units=pyo.units.kg,
-        doc="Reference maximum value of GAC mass needed for initial charge where "
+        doc="reference maximum value of GAC mass needed for initial charge where "
         "economy of scale no longer discounts the unit price",
     )
-    blk.adsorbent_unit_cost_coeff = pyo.Var(
-        initialize=4.58342,
-        units=pyo.units.USD_2020 * pyo.units.kg**-1,
-        doc="GAC adsorbent exponential cost pre-exponential coefficient",
-    )
-    blk.adsorbent_unit_cost_exp_coeff = pyo.Var(
-        initialize=-1.25311e-5,
-        units=pyo.units.kg**-1,
-        doc="GAC adsorbent exponential cost parameter coefficient",
-    )
-    blk.pres_other_cost_coeff = pyo.Var(
-        initialize=16660.7,
-        units=pyo.units.USD_2020,
-        doc="GAC pressure other cost power law coefficient",
-    )
-    blk.pres_other_cost_exp = pyo.Var(
-        initialize=0.552207,
+
+    # ---------------------------------------------------------------------
+    # correlation parameter data
+
+    # dummy data that is refixed in cost_gac based on the ContactorType
+    contactor_cost_coeff_dummy = {0: 10000, 1: 1000, 2: -10, 3: 0.1}
+    adsorbent_unit_cost_coeff_dummy = {0: 1, 1: -1e-5}
+    other_cost_param_dummy = {0: 10000, 1: 0.1}
+    energy_consumption_coeff_dummy = {0: 1e-3, 1: 1e-3, 2: 0}
+
+    # USD_2020 embedded in equation
+    blk.contactor_cost_coeff = pyo.Var(
+        range(len(contactor_cost_coeff_dummy)),
```

Comment:
You can just pass in the dictionary directly, or use `.keys()` if you want to be explicit, to set the indices. E.g.,
```suggestion
        contactor_cost_coeff_dummy,
```